public_configs/supported_payloads/Nix_vm.json

```diff
@@ -0,0 +1,28 @@
+[
+  {"name":"linux/x64/meterpreter/bind_tcp", "opts": []},
+  {"name":"linux/x64/meterpreter/reverse_tcp", "opts": []},
+  {"name":"linux/x64/meterpreter_reverse_http", "opts": []},
+  {"name":"linux/x64/meterpreter_reverse_https", "opts": []},
+  {"name":"linux/x64/meterpreter_reverse_tcp", "opts": []},
+  {"name":"linux/x86/meterpreter/reverse_tcp_uuid", "opts": []},
+  {"name":"linux/x86/meterpreter_reverse_http", "opts": []},
+  {"name":"linux/x86/meterpreter_reverse_https", "opts": []},
+  {"name":"linux/x86/meterpreter_reverse_tcp", "opts": []},
+  {"name":"java/meterpreter/bind_tcp", "opts": []},
```

Comment:
Looking at this list I am inclined to think the `nightly_tests.json` segment should just include `java.json` and `python.json` reducing the list here to just `linux/*` items.